modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -780,8 +780,24 @@ bool CvCapture_FFMPEG::open( const char* _filename )
     ic->interrupt_callback.opaque = &interrupt_metadata;
 #endif
 
-#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(52, 111, 0)
+#if LIBAVFORMAT_BUILD > CALC_FFMPEG_VERSION(51,11,0)
```

Comment:
With regards to 53.4.0, that makes sense to me. However, I am already making mistakes with regards to ffmpeg version checks, so ideally I would make as few changes as possible.